src/data_management/rasters/Raster.js

```diff
@@ -517,12 +513,12 @@ class Raster extends Component {
                   : `${buttonStyles.Button} ${buttonStyles.Inactive}`
               }
               onClick={this.handleFlushDataRasterClick}
-              style={{ maxHeight: "36px", width: "204px",marginRight: "10px" }}
+              style={{ maxHeight: "36px", width: "213px",marginRight: "10px" }}
```

Comment:
I think you can make it a bit bigger because for now, if I select 10 rasters then the text will go slightly off the button

Reply:
Thank you. :)